### PR TITLE
Improve checks and usage of environment variables

### PR DESCRIPTION
**Pull request by keesvanbochove**
For a new user coming into testing of this package, it's tricky to get the environment variables right. This adds checks to see if the expected environmental variables exist, and also hard codes the cohort_diagnostics schema name as Martijn suggested in #481

### CODE REVIEW

File: extras/PackageMaintenance.R [hunk 1]
```diff
@@ -82,7 +82,7 @@ connectionDetails <- DatabaseConnector::createConnectionDetails(dbms = "postgres
                                                                 user = Sys.getenv("CDM5_POSTGRESQL_USER"),
                                                                 password = URLdecode(Sys.getenv("CDM5_POSTGRESQL_PASSWORD")),
                                                                 server = Sys.getenv("CDM5_POSTGRESQL_SERVER"))
-cohortDiagnosticsSchema <- Sys.getenv("CDM5_POSTGRESQL_COHORT_DIAGNOSTICS_SCHEMA")
+cohortDiagnosticsSchema <- "cohort_diagnostics"
```
Comment (gowthamrao) [inline extras/PackageMaintenance.R:85]: @keesvanbochove  why not check if the environment variable exists. If it returns NULL then pass the hardcoded value as a fall back default value?

File: tests/testthat/setup.R [hunk 2]
```diff
@@ -26,9 +45,9 @@ if (runDatabaseTests) {
   
   cdmDatabaseSchema <- "eunomia"
   vocabularyDatabaseSchema <- "eunomia"
-  cohortDiagnosticsSchema <- Sys.getenv("CDM5_POSTGRESQL_COHORT_DIAGNOSTICS_SCHEMA")
+  cohortDiagnosticsSchema <- "cohort_diagnostics"
```
Comment (gowthamrao) [inline tests/testthat/setup.R:48]: see comment above - please change from hard coded

**Review by gowthamrao — CHANGES_REQUESTED** [review]
@keesvanbochove  could you make these minor changes please

**Comment by keesvanbochove** [comment]
OK, I see you made those too in the last hour, I've merged those in. Almost the same but I had forgotten to update PackageMaintenance.R.

**Pull request merged**